todoman/model.py

```diff
@@ -587,6 +589,8 @@ def todos(self, all=False, lists=[], urgent=False, location='',
             params.extend(lists)
         if urgent:
             extra_where.append('AND priority = 9')
+        elif priority:
+            extra_where.append('AND priority = ' + priority)
```

Comment:
If a user wants all tasks with medium priority, we show all the tasks that are of medium and high priority. Shouldn't we be just showing the tasks at the level 5 and not level <=5?